src/main/java/com/metamx/metrics/JvmThreadsMonitor.java

```diff
@@ -0,0 +1,58 @@
+package com.metamx.metrics;
+
+import com.google.common.base.Preconditions;
+import com.google.common.collect.ImmutableMap;
+import com.metamx.emitter.service.ServiceEmitter;
+import com.metamx.emitter.service.ServiceMetricEvent;
+import java.util.Map;
+import org.gridkit.lab.jvm.perfdata.JStatData;
+import org.gridkit.lab.jvm.perfdata.JStatData.LongCounter;
+
+public class JvmThreadsMonitor extends AbstractMonitor
```

Comment:
Maybe make this a part of `JvmMonitor`? I cannot imagine that we may want to monitor only threads, but not memery/GC stats in any of our services, or vice versa.

On the other hand, Monitors have to be explicitly enabled for each service, i. e. when this change is merged and a new version of server-metrics is released, you need not just update the dependency, but also update some configs. That is just extra work to do, and very easy to forget: just yesterday I found that we don't emit JVM CPU metrics in Druid, because somebody forgot to enable `JvmCpuMonitor` in the Druid config.